### PR TITLE
Refactor/UI profile and profile creation

### PR DESCRIPTION
**Pull request by GhaliBerbich**
The screens for profile, edit profile and create profile were synchronised to disply interests in the same form. Also added the edition of photo url in edit profile. And made the profile scrollable in order to access the elements at the bottom of the screen on a small phone.

### CODE REVIEW

File: app/src/androidTest/java/com/android/sample/ui/profile/ProfileCreationTest.kt
Comment (hmaryam84) [inline app/src/androidTest/java/com/android/sample/ui/profile/ProfileCreationTest.kt:1]: Removed testTags that are not accurate anymore and replace them with new ones that match the new features.

File: app/src/main/java/com/android/sample/ui/profile/EditProfile.kt
Comment (hmaryam84) [inline app/src/main/java/com/android/sample/ui/profile/EditProfile.kt:1]: LGTM: add the possibility of changing the photos with a new URL

File: app/src/main/java/com/android/sample/ui/profile/Profile.kt
Comment (hmaryam84) [inline app/src/main/java/com/android/sample/ui/profile/Profile.kt:1]: changed the lazy column configuration for a better responsivness (with small phones) 
the implementation LGTM

File: gradle/libs.versions.toml
Comment (hmaryam84) [inline gradle/libs.versions.toml:1]: not sure of this change

File: app/src/main/java/com/android/sample/ui/profile/ProfileCreation.kt
Comment (hmaryam84) [inline app/src/main/java/com/android/sample/ui/profile/ProfileCreation.kt:1]: the idea of Lazy Row is good and seems to be well implemented.

File: app/src/androidTest/java/com/android/sample/ui/profile/ProfileCreationTest.kt
Comment (hmaryam84) [inline app/src/androidTest/java/com/android/sample/ui/profile/ProfileCreationTest.kt:1]: Removing old inaccurate test tags and add new relevant ones

File: app/src/androidTest/java/com/android/sample/ui/profile/EditProfileTest.kt
Comment (hmaryam84) [inline app/src/androidTest/java/com/android/sample/ui/profile/EditProfileTest.kt:1]: the test is commented

**Review by hmaryam84 — COMMENTED** [review]
The new features seem to be well implemented, some unnecessary changes were made in the gradle file but the new push rolled back to the original gradle. The tests look like they verify all the test Tags and the new features approximatively well.

**Review by aminebenji04 — APPROVED** [review]
Thanks for adapting your code to all kinds of smartphones
This will work well.

**Pull request merged**